### ISSUE TITLE
MEI é empresário.

### ISSUE THREAD

**Issue by xexeo** [pilhacheia/tretadev#64]

**Comment by terremoth** [comment]
Sim.
Micro EMPRESA individual.

**Comment by callmarx** [comment]
Vota no Novo, acredita no estado minimo e em resiliência.

**Comment by jdoper** [comment]
Empresário malvadão!!!

**Comment by danielsobrinho** [comment]
> Sim.
> Micro EMPRESA individual.

Sim, assim como Eireli é Empresa individual, limitações pelo porte/faturamento e possibilidades atividades empresariais (cnaes).

**Comment by lincolixavier** [comment]
EXPLORADOR, ( de si mesmo )

**Comment by terremoth** [comment]
> EXPLORADOR, ( de si mesmo )

Minha opinião sobre isso é o seguinte, nunca vi MEI explorar a si mesmo, bem pelo contrário, o tanto que ele deixa o GOVERNO NÃO EXPLORÁ-LO é muito grande...

na CLT o governo pega pra ele entre 20% a 26% com IRRF + teto de INSS, se você ganhar mais de 4-5k por mes... No MEI você só paga 60 reais por mês e ali junto está incluído seu INSS  ([fonte](https://ingracio.adv.br/aposentadoria-mei/)).

Se você ganhar mais de 6 mil bruto de salário como dev CLT já nem vale mais a pena trabalhar de carteira assinada. Melhor ir de MEI mesmo combinando/acordando férias remuneradas e receber os benefícios da empresa. 

O senado aprovou uma lei que aumenta o limite anual do MEI de R$ 81 mil para R$ 130 mil. Isso quer dizer que como MEI, você pode conseguir tirar até R$10.833 por mês. ([fonte](https://folhabv.com.br/noticia/ECONOMIA/Economia/Projeto-aumenta-limite-de-faturamento-para-quem-deseja-se-tornar-MEI/79053))
"Ah mas e a categoria pra nota fiscal no MEI? [Usa-se qualquer um código do SPED na categoria de serviços de informática](http://sped.rfb.gov.br/pagina/show/1601), tem vários de programação, para você emitir na sua NFS-e

Fica bem claro pra mim que trabalhar como MEI (combinando e fazendo acordo com a empresa se possível por escrito, como um contrato de prestação de serviços por exemplo) é muito melhor que trabalhar como CLT. O governo basicamente tá sucateando o trabalhador. Acho que querem o fim da CLT zoando com ela? quem sabe...
